packages/faceted-search/package.json

```diff
@@ -5,22 +5,22 @@
   "mainSrc": "src/index.js",
   "license": "Apache-2.0",
   "scripts": {
-    "prepublish": "babel -d lib ./src/ && rimraf lib/**/*.test.js && cpx -v \"./src/**/*.scss\" ./lib",
-    "start": "start-storybook -p 6010",
-    "test": "jest --silent",
-    "test:noisy": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
+    "test": "cross-env TZ=Europe/Paris jest",
+    "test:watch": "cross-env TZ=Europe/Paris jest --watch",
+    "test:cov": "cross-env TZ=Europe/Paris jest --coverage",
+    "test:slimerjs": "slimerjs stories/slimer.js",
```

Comment:
It's not list in the dependencies
Provide documentation about it if we want to use it